coverage-config.yaml

```diff
@@ -0,0 +1,61 @@
+projects:
+  "zowe/perf-timing":
+    unit:
+      junit-artifact: perf-timing.yml/ubuntu-latest-16.x-results/unit/junit
+      lcov-artifact: perf-timing.yml/ubuntu-latest-16.x-results/unit/coverage
```

Comment:
is is possible to use `*ref` in yaml to hopefully parameterize this file.
I know this is just a simple search/replace-all operation for changing node versions.

---
After thinking through this a bit more, these packages could be updating their workflow node versions at different times